.github/workflows/ci-format.yml

```diff
@@ -94,7 +94,7 @@ jobs:
           . ${{env.venv_dir}}/bin/activate
           pip install --upgrade pip "setuptools<58.3.0"
           pip install wheel==0.38.4
-          pip install .[camera_obs,doc,train,ray]
+          pip install .[camera_obs,doc,train,ray,envision,argoverse,opendrive,waymo]
```

Comment:
In doc test, given we have `autodoc_mock_imports`, are the additional installs envision, argoverse, opendrive, waymo necessary?

Reply:
Yes, see https://github.com/huawei-noah/SMARTS/actions/runs/4875402139/jobs/8697543728#step:5:847